LogService/twisted/plugins/appscale_logserver_plugin.py

```diff
@@ -12,7 +12,7 @@
 class Options(usage.Options):
   optParameters = [["port", "p", 7422, "The port number to listen on."],
                    ["path", "a", "/opt/appscale/logserver", "Path where logs are stored."],
-                   ["size", "s", 25, "Size in GiB of retention of logs."],
+                   ["size", "s", 2, "Size in GiB of retention of logs."],
```

Comment:
That is correct. Currently GAE provides up to 2 GBs of logs.
A user can customize this value based on their from let's say the AppScalefile.

The root filesystem on any supported public cloud given will not be set up to hold that much data unless there are external volumes attached. This is a liability ATM.